packages/Modal/src/index.tsx

```diff
@@ -1,15 +1,21 @@
 import Button from "@blaze-react/button";
+import withUtils from "@blaze-react/utils";
 import React, { Fragment, useState } from "react";
+type TActions = [string, () => void, string?];
```

Comment:
Could we do an object inside in order to know when we use the component in the app what is each position. Something like [{textButton: string, callback: () => void, modifier: string}]